### PR TITLE
Sc/lti advantage

### PR DESCRIPTION
**Pull request by seanrcollings**

### CODE REVIEW

File: app/controllers/atomic_admin/atomic_tenant_deployment_controller.rb [hunk 1]
```diff
@@ -41,17 +35,20 @@ def search
...
+      page, meta = filter(
+        AtomicTenant::LtiDeployment.where(application_instance_id:),
+        search_col: "deployment_id",
+      )
+
+      render json: {
+        deployments: page,
+        meta:
+      }
+    end
```
Comment (seanrcollings) [inline app/controllers/atomic_admin/atomic_tenant_deployment_controller.rb:48]: @blunckr-aj I think you touched some of the LTI deployment code recently & the search method above this is more complicated than I made this. What's the reasoning for the additional complexity?

File: app/controllers/atomic_admin/atomic_tenant_client_id_strategy_controller.rb [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
-      params.permit(:iss, :client_id, :application_instance_id)
-    end
-
-    def find_pinned_client_id
-      AtomicTenant::PinnedClientId.find_by(id: params[:id])
-    end
+    include Filtering
 
-    def search 
+    def search
```
Comment (blunckr-aj) [inline app/controllers/atomic_admin/atomic_tenant_client_id_strategy_controller.rb:5]: maybe we could add a comment that this one is deprecated/only used by the old admin app. same with the other controllers

File: app/controllers/atomic_admin/atomic_tenant_deployment_controller.rb (hunk 1 above)
Comment (blunckr-aj) [reply app/controllers/atomic_admin/atomic_tenant_deployment_controller.rb:48]: We discussed in slack, but the original version pulls in info from atomic_tenant that we don't think really needs to be here

File: app/controllers/atomic_admin/atomic_tenant_client_id_strategy_controller.rb (hunk 2 above)
Comment (seanrcollings) [reply app/controllers/atomic_admin/atomic_tenant_client_id_strategy_controller.rb:5]: Good call

**Pull request merged**